examples/low-level-api/create_vc.rs

```diff
@@ -41,11 +44,11 @@ pub async fn create_vc() -> Result<()> {
   println!("Credential JSON > {:#}", credential);
 
   // Validate the verifiable credential
-  let validation: CredentialValidation = common::check_credential(&client, &credential).await?;
-  println!("Credential Validation > {:#?}", validation);
-  assert!(validation.verified);
-
-  Ok(())
+  let validator = CredentialValidator::new();
```

Comment:
Is this going to follow the builder pattern later?

Reply:
No. In fact all methods on `CredentialValidator` don't use any state currently, but we have made them take `&self` to open up for that possibility in the future in case we need it.